### ISSUE TITLE
specs in pending suites should be processed

### ISSUE THREAD

**Issue by snogge** [jorgenschaefer/emacs-buttercup#116]
Any suites and specs nested in a xdescribe form will be skipped during processing and never added to buttercup-suites.  This means they are not counted and not matchable with patterns.

For instance, the command

```ShellSession
emacs -batch -L . -l buttercup.el -f buttercup-run-markdown docs/writing-tests.md
```

in the emacs-buttercup folder will print 

    Ran 50 out of 52 specs, 0 failed, in 0.0 seconds.

while there is actually a 53rd spec at line 318 of writing-tests.md that is skipped and not counted.

If a matching pattern should match any spec in a xdescribe form, that spec will not be found and the containing, skipped, xdescribe will not be run either.

**Comment by jorgenschaefer** [comment]
Yeah … `xdescribe` is basically a big no-op form. I'm not sure if this is useful enough to warrant the added complexity of supporting it. So, backlog here, nothing I'll put any kind of focus on.

**Comment by snogge** [comment]
Just wanted to be sure this is not a behavior you want because that is how jasmine works.

**Comment by jorgenschaefer** [comment]
I think including this feature would be nice if it does not make other parts of the code less readable, if that helps :-D
